src/docs/development/ui/advanced/slivers.md

```diff
@@ -1,19 +1,66 @@
 ---
-title: Slivers
-description: What are slivers in Flutter and how to use them.
+title: Using slivers to achieve fancy scrolling
+description: Where to find information on using slivers to implement fancy scrolling effects, like elastic scrolling, in Flutter.
```

Comment:
Maybe add `toc: false`? It's a little weird to have a TOC with one item in it. Not a big deal, though.